network-api/networkapi/wagtailpages/pagemodels/base.py

```diff
@@ -126,7 +126,7 @@ class InitiativesPage(PrimaryPage):
     ]
 
 
-class ParticipatePage2(PrimaryPage):
+class ParticipatePage(PrimaryPage):
```

Comment:
This is the only "true" change, everything else is just changes to make sure references to the old name are now references to the new name.